package.json

```
@@ -27,14 +27,19 @@
     "test": "npx hardhat test",
     "coverage": "npx hardhat coverage",
     "format": "npx prettier --write",
-    "check-format": "prettier --ignore-path .gitignore --check '{scripts,test,contracts}/**/*.{js,sol}'",
+    "check-format": "prettier --check '{scripts,test,contracts}/**/*.{js,sol}'",
     "check-types": "tsc --noEmit",
     "doc": "npx hardhat docgen",
-    "uml": "./scripts/utils/sol2uml.sh"
+    "uml": "./scripts/utils/sol2uml.sh",
+    "lint:sol": "prettier --log-level warn '{contracts,test}/**/*.sol' --check && solhint '{contracts,test}/**/*.sol'",
+    "lint:sol:fix": "prettier --log-level warn '{contracts,test}/**/*.sol' --write"
```

I was suggesting soemthing like thatthat:
```suggestion
    "lint:sol": "prettier --log-level warn 'contracts/**/*.sol' --check && solhint 'contracts/**/*.sol'",
    "lint:sol:fix": "prettier --log-level warn 'contracts/**/*.sol' --write"
```